lib/ipmi_backend_utils.pm

```diff
@@ -36,6 +37,24 @@ sub use_ssh_serial_console {
     bmwqemu::save_vars();
 }
 
+sub switch_from_ssh_to_sol_console {
+    my (%opts) = @_;
+    my $reset_console_flag = $opts{'reset_console_flag'};
+
+    #close root-ssh console
+    prepare_system_shutdown;
+    #switch to sol console
+    set_var('SERIALDEV', '');
+    $serialdev = 'ttyS1';
+    bmwqemu::save_vars();
+    console('sol')->disable;
+    if ($reset_console_flag eq "on") {
```

Comment:
As it's only usage, why not to use $opts{reset_console_flag} here? We don't need new verification run in case you will change this.

Reply:
Fine, will change to it.